config/webpack/terra-dev-site.webpack.config.js

```diff
@@ -79,25 +103,49 @@ const devSiteConfig = (env = {}, argv = {}) => {
     console.log('Generated Aliases', alias);
   }
 
+  const indexEntryPoints = [];
+  indexEntryPoints.push('terra-dev-site');
+  // Strip the trailing / from the public path.
+  const basename = publicPath.slice(0, -1);
+
   return {
     entry: {
       'terra-dev-site': path.resolve(path.join(__dirname, '..', '..', 'lib', 'Index')),
+      rewriteHistory: path.resolve(path.join(__dirname, '..', '..', 'lib', 'rewriteHistory')),
+      redirect: path.resolve(path.join(__dirname, '..', '..', 'lib', 'redirect')),
     },
     plugins: [
-      new HtmlWebpackPlugin({
+      // terra-dev-site index.html
+      indexPlugin({
         title: siteConfig.appConfig.title,
-        template: path.join(__dirname, '..', '..', 'lib', 'index.html'),
+        filename: 'index.html',
         lang,
-        dir: siteConfig.appConfig.defaultDirection,
-        favicon: siteConfig.appConfig.favicon,
-        headHtml: [getNewRelicJS()].concat(siteConfig.appConfig.headHtml),
-        inject: false, // this turns off auto injection. We handle this ourselves in the template.
+        siteConfig,
+        indexEntryPoints,
+        entry: 'terra-dev-site',
```

Comment:
Can we just hard-code this in the entry points filter above?

Reply:
What you're not seeing is that i'm going to reuse this function in the future to generate multiple sites for Orion dev site